ui/apps/platform/src/Containers/ComplianceEnhanced/Schedules/Wizard/useFormikScanConfig.tsx

```diff
@@ -46,6 +54,25 @@ const validationSchema = yup.object().shape({
     }),
     clusters: yup.array().min(1),
     profiles: yup.array().min(1),
+    report: yup.object().shape({
+        notifierConfigurations: yup
+            .array()
+            .of(
+                yup.object().shape({
```

Comment:
I imagine you considered adding this shape to `EmailTemplate.utils.ts`, curious why you decided against that? That's not to say I would've done it any differently.

Reply:
Aha, that was a mental fail. I did not because of the extra condition in vulnerability reports, but it is indeed possible to factor out the common part.
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Containers/Vulnerabilities/VulnerablityReporting/forms/useReportFormValues.tsx#L135-L145

Also good that you direct my attention to validation, because it deserves careful testing, especially after you found the edge case in mailing lists.